src/Components/AllComents.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+import "../comments.css";
+
+function AllComments() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(
```

Comment:
It would be better to extract all our api calls out into an api.js file in root to tidy up our components and allow them all to be configured in one place.